src/aqua/installation-spell/src/aqua/deploy.aqua

```diff
@@ -63,7 +57,8 @@ func deploy(config: AppDeployDefinition) -> DeployedAppWorkers:
                     config.installation_script,
                     config.installation_trigger,
                     w.name,
-                    w.definition
+                    w.definition,
+                    w.dummy_deal_id
                 )
                 spells <<- DeployedSpell(spell_id = spell_id, worker_id = worker_id, host_id = h)
                 on INIT_PEER_ID via HOST_PEER_ID:
```

Comment:
Can you please also add dummy_deal_id to deployed worker defenition so I have it conveniently after deploy so I can store it along everything else